drizzlepac/util.py

```diff
@@ -376,14 +381,14 @@ def _ptime():
     if dtime:
         # This time stamp includes sub-second timing...
         _ltime = dtime.datetime.fromtimestamp(ftime)
-        tlm_str = _ltime.strftime("%H:%M:%S")+str(_ltime.microsecond/1e+6)[1:-3]+_ltime.strftime(" (%d/%m/%Y)")
+        tlm_str = _ltime.strftime("%H:%M:%S" + str(_ltime.microsecond / 1e+6)[1:-3] + _ltime.strftime(" (%d/%m/%Y)"))
     else:
         # Basic time stamp which only includes integer seconds
         # Format time values for keywords IRAF-TLM, and DATE
         _ltime = time.localtime(ftime)
-        tlm_str = time.strftime('%H:%M:%S (%d/%m/%Y)',_ltime)
-        #date_str = time.strftime('%Y-%m-%dT%H:%M:%S',_ltime)
-    return tlm_str,ftime
+        tlm_str = time.strftime('%H:%M:%S (%d/%m/%Y)', _ltime)
+        # date_str = time.strftime('%Y-%m-%dT%H:%M:%S',_ltime)
+    return tlm_str, ftime
```

Comment:
Insert space before _ltime at line 391 although this line of code seems to be deprecated and could be deleted.

Reply:
Line 390 was deleted in b11537b4 (just to keep things simpler...)